lib/financial_period.rb

```diff
@@ -21,4 +22,30 @@ def self.year_from_date(date)
     end
     (year - 1).to_s
   end
+
+  def self.current_quarter_string
+    quarter_from_date(Date.today).to_s
+  end
+
+  def self.current_year_string
+    year_from_date(Date.today).to_s
+  end
+
+  def self.next_ten_years
+    this_year = Date.today.year
+    tenth_year = this_year + 9
+    (this_year..tenth_year).step.to_a
```

Comment:
Quick question... So if you don't specify anything between brackets after `step`, Ruby will default to one whole number at a time, right?

Reply:
Yep, the step here is a method call on the `(this_year..tenth_year)` which is a Range, so basically `Range.step` the docs for Range are here https://ruby-doc.org/core-2.6.2/Range.html#method-i-step you can see that when nothing is passed, step simply calls `succ` which is the equivilent of +1 each time. 

It probably works with `each` do you think that would improve this?

Reply:
I think using `each` wouldn't make much difference. I like having `step` here (I learned about it thanks to this line of code)